scripts/build-graphql-schema.ts

```diff
@@ -0,0 +1,17 @@
+import yargs from 'yargs'
```

Comment:
I would probably keep the same name between the script and the command name inside the workspace

Reply:
agree!